server/routes/index.ts

```diff
@@ -8,6 +8,8 @@ import RedisTokenStore from '../data/tokenStore/redisTokenStore'
 import { createRedisClient } from '../data/redisClient'
 import { IncidentReportingApi } from '../data/incidentReportingApi'
 import config from '../config'
+import ManageUsersApiClient, { type User } from '../data/manageUsersApiClient'
```

Comment:
I think the linter is complaining about the `User` not being used (so you can probably remove the import?)

Reply:
removed it now. think I needed it at some point when I was putting stuff together